Cargo.toml

```diff
@@ -11,8 +11,8 @@ readme = "README.md"
 vendored = ["openssl/vendored"]
 
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
-security-framework = "0.2.1"
-security-framework-sys = "0.2.1"
+security-framework = "=0.2.1"
```

Comment:
= dependencies cause significant issues resolving a dependency graph, and shouldn't be used.